healthcheck/healthcheck.go

```diff
@@ -42,6 +42,8 @@ func MonitorExternal(clients ...Client) {
 	for _, client := range clients {
 		go func(client Client) {
 			if name, err := client.Healthcheck(); err != nil {
+				err := fmt.Errorf("unsuccessful healthcheck for %s: %v", name, err)
```

Comment:
I thought we deprecated use of `fmt.Errorf` (makes searches on kibana more tricky), how about:
```
log.ErrorC(""unsuccessful healthcheck", err, log.Data{"external_service": name})
```

Reply:
Done 👍